GUI/gui.py

```diff
@@ -0,0 +1,197 @@
+
+chngmac    - change mac addresses of all interfaces
+
+rvmac      - revert mac addresses of all interfaces
+
+version    - print version of torctl and exit"""
+
+ABOUT = """Official Repository: blackarch.org
+
+GUI Developer: Aleff (github.com/aleff-github)
+
+Licence: GPLv3"""
+
+CONTACT = """Github: https://github.com/aleff-github
+
+Instagram: https://www.instagram.com/alessandro_greco_aka_aleff/
+
+Linkeding: https://www.linkedin.com/in/alessandro-greco-aka-aleff/"""
+
+def tmp():
+    print("ok")
+
+def get_update_lines(output: str):
+    lines = output.decode("utf-8").splitlines()
+    for line in lines:
+        if re.search(r"^\[", line):
+            info_panel.insert(tk.END, line+"\n")
+    info_panel.see(tk.END)
+
+def initial_state():
+    get_update_lines(subprocess.check_output("sudo torctl ip", shell=True))
+    get_update_lines(subprocess.check_output("sudo torctl status", shell=True))
+
+def start():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nSTART\n")
+    get_update_lines(subprocess.check_output("sudo torctl start", shell=True))
+    info_panel.configure(state="disabled")
+
+def stop():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nSTOP\n")
+    get_update_lines(subprocess.check_output("sudo torctl stop", shell=True))
+    info_panel.configure(state="disabled")
+
+def status():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nSTATUS\n")
+    get_update_lines(subprocess.check_output("sudo torctl status", shell=True))
+    info_panel.configure(state="disabled")
+
+def restart():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nRESTART\n")
+    get_update_lines(subprocess.check_output("sudo torctl restart", shell=True))
+    info_panel.configure(state="disabled")
+
+def autowipe():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nAUTO WIPE\n")
+    get_update_lines(subprocess.check_output("sudo torctl autowipe", shell=True))
+    info_panel.configure(state="disabled")
+
+def autostart():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nAUTO START\n")
+    get_update_lines(subprocess.check_output("sudo torctl autostart", shell=True))
+    info_panel.configure(state="disabled")
+
+def ip():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nWHAT IS MY IP?\n")
+    get_update_lines(subprocess.check_output("sudo torctl ip", shell=True))
+    info_panel.configure(state="disabled")
+
+def chngid():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nCHANGE TOR IDENTITY\n")
+    get_update_lines(subprocess.check_output("sudo torctl chngid", shell=True))
+    info_panel.configure(state="disabled")
+
+def chngmac():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nCHANGE MAC\n")
+    get_update_lines(subprocess.check_output("sudo torctl chngmac", shell=True))
```

Comment:
Use absolute paths instead of "sudo command"

Reply:
Is this valid for any command?

Reply:
Replaced the `sudo` command with `/usr/local/bin/torctl`